_posts/2018-10-01-why-developers-do-not-like-change.md

```diff
@@ -0,0 +1,75 @@
+---
+author: Sam Davies
+layout: post
+asset-type: post
+title: "Why developers do not like organisational change"
+date: 2018-10-01 00:00:00
+description: A glimpse into the mind of developers when organisational changes are proposed.
+image: 
+    src: /assets/custom/img/blog/2018-10-01-architecture.jpg
+abstract: A glimpse into the mind of developers when organisational changes are proposed.
+---
+This post will endeavour to explore the reasons why developers do not like organisational change, and how not
+recognising this could pose a big problem for your company. 
+
+## No one likes change
+Let me start off by saying that we as humans are not big advocates of change. To illustrate this, if I told you to move
+house every week, you would probably not be overly enamoured by the idea. We seek stability the majority of the time.
+
+This is not to say we do not seek change from time to time:
+
+ * Move house
+ * Change job
+ * Re-decorate a room
+ * Change your car
+
+## No one likes to lose
+From birth, it is pre-determined that we do not like losing. In the brilliant book 
+[Thinking, Fast and Slow](https://www.amazon.co.uk/Thinking-Fast-Slow-Daniel-Kahneman/dp/0141033576) talks about gains
```

Comment:
Should "In the brilliant book Thinking, Fast and Slow talks about..." be "In the brilliant book Thinking, Fast and Slow Daniel Kaheman talks about..."